.github/workflows/neuron-ci.yml

```diff
@@ -141,7 +141,7 @@ jobs:
         run: |
           sudo apt-get install build-essential ccache libopenmpi-dev \
             libmpich-dev libx11-dev libxcomposite-dev mpich ninja-build \
-            openmpi-bin flex libfl-dev bison
+            openmpi-bin flex libfl-dev bison libreadline-dev
```

Comment:
if an explicit install for readline is needed then we should mention in https://github.com/neuronsimulator/nrn/blob/master/docs/install/install_instructions.md#linux-1 ?